bin/hdfcoinc/pycbc_fit_sngls_over_multiparam

```diff
@@ -0,0 +1,129 @@
+#!/usr/bin/python
+
+# Copyright 2016 Thomas Dent, Alex Nitz
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of the GNU General Public License as published by the
+# Free Software Foundation; either version 3 of the License, or (at your
+# option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General
+# Public License for more details.
+
+from __future__ import division
+
+import sys, h5py, argparse, logging, pycbc.version, numpy
+from pycbc.events import trigger_fits as trstats
+from scipy import interpolate
+from scipy.ndimage.filters import gaussian_filter
+
+parser = argparse.ArgumentParser(usage="",
+    description="Smooth (regress) the dependence of coefficients describing "
+                "single-ifo background trigger distributions on a template "
+                "parameter, to suppress random noise in the resulting "
+                "background model.")
+
+parser.add_argument("--version", action=pycbc.version.Version)
+parser.add_argument("-V", "--verbose", action="store_true",
+                    help="Print extra debugging information", default=False)
+parser.add_argument("--template-fit-file",
+                    help="Inumpyut hdf5 file containing fit coefficients for each"
+                         " individual template. Required")
+parser.add_argument("--bank-file", default=None,
+                    help="hdf file containing template parameters. Required "
+                         "unless reading param from template fit file")
+parser.add_argument("--output", required=True,
+                    help="Location for output file containing smoothed fit "
+                         "coefficients.  Required")
+parser.add_argument("--use-template-fit-param", action="store_true",
+                    help="Use parameter values stored in the template fit file"
+                         "as template_param for smoothing.", default=False)
+parser.add_argument("--fit-param", nargs='+',
+                    help="Parameter over which to regress the background "
```

Comment:
Should rewrite the help here to emphasize that several parameters may be passed and also display the new choices (eta and chi_eff)